src/ingest_validation_tools/table-schemas/level-2/seqfish.yaml

```diff
@@ -1,5 +1,23 @@
 doc_url: https://docs.google.com/document/d/1H_z5QQvXP-5GKwiKRF2GhcpmqEUDWYnoOIS---Uepas/edit
 fields:
+  # Overrides of Level 1 fields:
+  -
+    name: assay_category
+    constraints:
+      enum:
+        - sequence
```

Comment:
imaging

Reply:
I'm doing `imaging` instead of `sequence`, not in addition to, right?